### ISSUE TITLE
Focus indicator in blau button secondary is not visible

### ISSUE THREAD

**Issue by aweell** [Telefonica/mistica-design#2613]
## Description

https://jira.tid.es/browse/ANDROID-17649

Current definition of the focus indicator in buttonSecondaryInverse using its border token is causing Blau focus to be invisible, since button secondary inverse variant in Blau brand has no border but background.
 

## Tasks

- [x] [Update specs](https://www.figma.com/design/koROdh3HpEPG2O8jG52Emh/branch/nupBXOiyvymVxmqmHfM1OI/%F0%9F%94%B8-Buttons-Specs?m=auto&t=fgzBF0bOSiGpYa3V-1)
- [ ] Design documentation
- [ ] Add to mobile library
- [ ] Add to desktop library
- [ ] Add to UI Kit
- [ ] Web ticket

**Comment by tuentisre** [comment]
Figma branches linked to this issue

| File | Branch | Last Modification |
| --- | --- | --- |
| 🔸 Buttons Specs | [#2613 - Focus indicator in blau button secondary is not visible](https://www.figma.com/file/koROdh3HpEPG2O8jG52Emh/branch/nupBXOiyvymVxmqmHfM1OI) | 1 d |
